include/xkbcommon/xkbcommon.h

```diff
@@ -3456,6 +3456,87 @@ xkb_state_led_index_is_active(struct xkb_state *state, xkb_led_index_t idx);
 
 /** @} */
 
+/**
+ * @defgroup test-api Features availability
+ * Test features availability
+ *
+ * @{
+ */
+
+/**
+ * @enum xkb_enumeration
+ *
+ * Enumerate all the enumerations testable with `xkb_enumeration_is_valid()`.
+ *
+ * @since 1.14.0
+ */
+enum xkb_enumeration {
+    /** @since 1.14.0 */
+    XKB_ENUM_ENUMERATION = 0,
+    /** @since 1.14.0 */
+    XKB_ENUM_RMLVO_BUILDER_FLAGS,
```

Comment:
Two things:
- I strongly recommend writing these out as individual values instead of just relying on the order. It will make it much more obvious when a value changes accidentally.
- I recommend starting the enum values at e.g. 100 so there cannot be accidental overlap (and false positives) between `XKB_FOO_FLAGS` and its corresponding `XKB_ENUM_FOO_FLAGS`.